### ISSUE TITLE
Error when stopping and restarting an app from "Etats"

### ISSUE THREAD

**Issue by ZeHiro** [cozy/cozy-home#469]
Maybe it is a controller problem.

Hello, I have regularly this can of error in the controller log (self hosted instance) : 
```

ications.js:129:13                                                              
    at Object.module.exports.stop (/usr/local/lib/node_modules/cozy-controller/b
uild/server/lib/controller.js:284:12)                                           
    at Object.module.exports.stop [as handle] (/usr/local/lib/node_modules/cozy-
controller/build/server/controllers/applications.js:126:21)                     
    at next_layer (/usr/local/lib/node_modules/cozy-controller/node_modules/amer
icano/node_modules/express/lib/router/route.js:103:13)                          
    at Object.module.exports.check [as handle] (/usr/local/lib/node_modules/cozy
-controller/build/server/middlewares/token.js:31:16)                            
    at next_layer (/usr/local/lib/node_modules/cozy-controller/node_modules/amer
icano/node_modules/express/lib/router/route.js:103:13)                          
    at Route.dispatch (/usr/local/lib/node_modules/cozy-controller/node_modules/
americano/node_modules/express/lib/router/route.js:107:5)                       
    at /usr/local/lib/node_modules/cozy-controller/node_modules/americano/node_m
odules/express/lib/router/index.js:213:24                                       
    at param (/usr/local/lib/node_modules/cozy-controller/node_modules/americano
/node_modules/express/lib/router/index.js:303:14)                               
    at param (/usr/local/lib/node_modules/cozy-controller/node_modules/americano
/node_modules/express/lib/router/index.js:319:14)         

```

**Comment by clochix** [comment]
Thanks for reporting, could you detail the steps to reproduce this error ?

**Comment by ZeHiro** [comment]
Is definitely a controller problem. I close and reopen in the right repo.

**Comment by ZeHiro** [comment]
Ok I think I get it. 
And it is a home problem :) 
Action to reproduce :  
- From "Etats" in home, stop an application
- When the application is stopped, "Start the application" is proposed to the user. Clicke start. The client sends request "POST /drones/APP/stop" instead of start. The app finally restarts automatically.

**Comment by clochix** [comment]
Hi @ZeHiro,
This should have been fixed, could you update the controller and check if it now works as you expect?

**Comment by poupotte** [comment]
This issue should be fixed since  	f6c0fd3. I close this issue, @ZeHiro , if you have still the problem, don't heistate to reopen it.
